pkg/extension/opampextension/testdata/opamp.d/opamp-filter-processor.yaml

```diff
@@ -0,0 +1,103 @@
+      operators:
+        - type: add
+          field: resource["_sourceName"]
+          value: System
+    windowseventlog/Microsoft-Windows-PowerShell/Operational/09c98ea4-6686-4ab9-b3cd-f04070906441:
+      channel: Microsoft-Windows-PowerShell/Operational
+      operators:
+        - type: add
+          field: resource["_sourceName"]
+          value: Microsoft-Windows-PowerShell/Operational
+    windowseventlog/Amazon EC2Launch/09c98ea4-6686-4ab9-b3cd-f04070906441:
+      channel: Amazon EC2Launch
+      operators:
+        - type: add
+          field: resource["_sourceName"]
+          value: Amazon EC2Launch
+  processors:
+    memory_limiter:
+      check_interval: 5s
+      limit_percentage: 75
+      spike_limit_percentage: 20
+    batch:
+      send_batch_size: 1_024
+      timeout: 1s
+    resourcedetection/system:
+      detectors: ["system"]
+      system:
+        hostname_sources: ["dns","os"]
+    resource/metrics/09c98ea4-6686-4ab9-b3cd-f04070906441:
+      attributes:
+      - key: _sourceName
+        value: windows
+        action: insert
+    resource/09c98ea4-6686-4ab9-b3cd-f04070906441:
+      attributes:
+      - key: _source
+        value: sample_win_st
+        action: insert
+      - key: _contentType
+        value: OpenTelemetry
+        action: insert
+      - key: sumo.datasource
+        value: windows
+        action: insert
+    filter/logs/09c98ea4-6686-4ab9-b3cd-f04070906441:
+      logs:
+        log_record:
+          - not(IsMatch(body, ".*\"id\":40962.*") or IsMatch(body, ".*\"id\":8197.*"))
+          - IsMatch(body, ".*\"id\":8197.*")
+  exporters:
+    sumologic:
+      sending_queue:
+        enabled: true
+        storage: file_storage
+      retry_on_failure:
+        max_elapsed_time: 0
+  service:
+    pipelines:
+      metrics/windows/09c98ea4-6686-4ab9-b3cd-f04070906441:
+        receivers:
+          - hostmetrics/09c98ea4-6686-4ab9-b3cd-f04070906441
+        processors:
+          - memory_limiter
+          - resourcedetection/system
+          - batch
+          - resource/metrics/09c98ea4-6686-4ab9-b3cd-f04070906441
+          - resource/09c98ea4-6686-4ab9-b3cd-f04070906441
+        exporters:
+          - sumologic
+      logs/windows/09c98ea4-6686-4ab9-b3cd-f04070906441:
+        receivers:
+          - windowseventlog/Application/09c98ea4-6686-4ab9-b3cd-f04070906441
+          - windowseventlog/Security/09c98ea4-6686-4ab9-b3cd-f04070906441
+          - windowseventlog/System/09c98ea4-6686-4ab9-b3cd-f04070906441
+          - windowseventlog/Microsoft-Windows-PowerShell/Operational/09c98ea4-6686-4ab9-b3cd-f04070906441
+          - windowseventlog/Amazon EC2Launch/09c98ea4-6686-4ab9-b3cd-f04070906441
+        processors:
+          - memory_limiter
+          - resourcedetection/system
+          - batch
+          - resource/09c98ea4-6686-4ab9-b3cd-f04070906441
+          - filter/logs/09c98ea4-6686-4ab9-b3cd-f04070906441
+        exporters:
+          - sumologic
```

Comment:
Do we need all of this configuration just to test filterprocessor?

Reply:
this is an example source template from the backend team, that I thought would be useful to run a test against.

Reply:
I feel like it is more than needed. Personally I would like to limit test scope to what is necessary, eventually we can add another test which will be more complex, but now most of the configuration seems to be unrelated to the name of the test/file

Reply:
If the test fail, it won't be obvious why

Reply:
ok I'll trim some portions of the config.

Reply:
how does this look - https://github.com/SumoLogic/sumologic-otel-collector/pull/1547/commits/7fcad3e5406cebd3fb5e8bf231ab66953b3f8851

it has 1 receiver, 2 filter processors, 1 transform processor, 1 exporter.

Reply:
Yup, thats better, LGTM :) (I marked batch processor, you can clean it up if unnecessary)